_posts/2020-02-03-release-1.10.0.md

```diff
@@ -0,0 +1,204 @@
+---
+layout: post 
+title:  "Apache Flink 1.10.0 Release Announcement" 
+date: 2020-02-03T02:30:00.000Z
+categories: news
+authors:
+- morsapaes:
+  name: "Marta Paes"
+  twitter: "morsapaes"
+---
+
+
+The Apache Flink community is excited to hit the double digits and announce the release of Flink 1.10.0! As a result of the biggest community effort to date, with over 1.2k issues implemented and 218 contributors, this release introduces significant improvements to the overall performance and stability of Flink jobs, a preview of native Kubernetes integration and great advances in Python support (PyFlink). 
+
+Flink 1.10 also marks the completion of the [Blink integration](https://flink.apache.org/news/2019/08/22/release-1.9.0.html#preview-of-the-new-blink-sql-query-processor), hardening streaming SQL and bringing mature batch processing to Flink with production-ready Hive integration and TPC-DS coverage. This blog post describes all major new features and improvements, important changes to be aware of and what to expect moving forward.
+
+{% toc %}
+
+The binary distribution and source artifacts are now available on the updated [Downloads page]({{ site.baseurl }}/downloads.html) of the Flink website. For more details, check the complete [release changelog]() and the [updated documentation]({{ site.DOCS_BASE_URL }}flink-docs-release-1.10/). We encourage you to download the release and share your feedback with the community through the [Flink mailing lists](https://flink.apache.org/community.html#mailing-lists) or [JIRA](https://issues.apache.org/jira/projects/FLINK/summary).
```

Comment:
@morsapaes It seems there's still an empty log?

Reply:
https://flink.apache.org/downloads.html will be updated by #302 accordingly.

Reply:
The link for `release changelog` is truly missing.

Reply:
Here is the link for the changelog, FYI:
https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12315522&version=12345845

Reply:
Thanks for spotting it, @carp84 and @xintongsong!